README.md

```diff
@@ -24,6 +24,16 @@ See the videos and how many times you watched them. The links are clickable:
 
 NOTE: In the test sample above, it looks like "Music Artist" is duplicated, but that is because the video links are different! One link points to <a href="https://music.youtube.com">music.youtube.com</a> and the other points to <a href="https://www.youtube.com">www.youtube.com</a>. So you you get separate counts for how many times you viewed a video versus listened to it.
 
+## How to build and run from source
```

Comment:
just some notes based on the lack of info on what environment to use or how to run it direct from source.